gamemodes/terrortown/gamemode/server/sv_voice.lua

```diff
@@ -30,9 +30,15 @@ local cv_ttt_limit_spectator_voice = CreateConVar("ttt_limit_spectator_voice", "
 -- stylua: ignore
 local loc_voice = CreateConVar("ttt_locational_voice", "0", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
 
+---
+-- @realm server
+-- stylua: ignore
+local loc_voice_prep = CreateConVar("ttt_locational_voice_prep", "0", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
```

Comment:
imho, I'd change this to include post as well, don't you think? During post spectators shold also be able to speak witrh everyone again imho (which is probably already the case?)

Reply:
yes, that is already the case and i don't think it needs to be configurable.

Reply:
I was talking about the locational voice. It should be disabled in post as well, if it is disabled in prep. At least in my opinion. No secondary convar, this one should handle both

Reply:
It is already disabled in post and i don't think that needs changing. Especially not depending on if locational voice is disabled in prep.

Reply:
Locational voice is already disabled in post? I wasn't aware of that

Reply:
Then the UI should probably have a helpbox with "Locational voice is always disabled during the post round phase" to make it a bit more clear

Reply:
maybe, but that's unrelated to this PR

Reply:
yeah, I find it hard to imagine a use case where you want it enabled during post anyway since all but a few players are probably spectators at that point, but it definitely shouldn't be linked to the preparing phase.